### DISCUSSION TITLE
Aggiornamento ESPHome

### DISCUSSION THREAD

**Discussion by sky7176** [indomus/forum#1100] [HUB: Home Assistant]
Buona sera a tutti,
ma vi è arrivato l'aggiornamento ESPHome 2021.11?

Io sono ancora fermo alla versione 2021.10.3, ho letto la recensione dell'uscita della nuova versione, ma per quanto mi riguarda non mi è ancora arrivata la notifica di aggiornamento.

Io uso Assio su RP4

Giuseppe

**Comment by indomus** [comment]
[Come vedi](https://hub.docker.com/r/esphome/esphome-hassio-aarch64/tags) è uscita. Questo è uno dei difettucci di HASSIO, a volte non vede gli aggiornamenti (cosa che non capita se installi Home Assitant Core in altro modo e le altre componenti te le gestisci in modo singolo)

**Reply by sky7176** [reply]
Ciao, grazie,
ho provato a riavviare ESPHome e mi segnala che l'addon è all'ultima versione disponibile.
Cercher info ulteriori.

Giuseppe

**Reply by Nik71git** [reply]
come vedi l'icona di ESPHome? se è quella di servizio (un pezzo di puzzle) allora devi controllare di avere nelle repository quella corretta, cioè https://addons.community; eventualmente lo trovi anche qui: https://github.com/esphome/esphome

**Reply by sky7176** [reply]
Ciao,
ho provato ad aggiungere i due repo, ma mi restituisce l'errore :"Invalid Add-on repository!"
per entrambi i link.
Si l'icone del servizio è il pezzo di puzzle

Giuseppe

**Reply by sky7176** [reply]
> 

Risolto,
ho applicato la prima regola dell'informatica: spegni  e riavvia
Ora mi ha dato l'aggiornamento

**Reply by Nik71git** [reply]
ottimo, comunque dovrebbe bastarti la prima repo.
